spacy/matcher/matcher.pyx

```diff
@@ -101,16 +103,15 @@ cdef class Matcher:
         key (str): The match ID.
         patterns (list): The patterns to add for the given key.
         on_match (callable): Optional callback executed on match.
-        *_patterns (list): For backwards compatibility: list of patterns to add
-            as variable arguments. Will be ignored if a list of patterns is
-            provided as the second argument.
+        greediness (str): "FIRST" or "LONGEST".
```

Comment:
or maybe some kind of `Enum` orso?

Reply:
I tried to use `Enum` for the Chinese tokenizer settings and it seemed like a total hassle so I went back to a flimsy tuple, but it's kind of unsatisfying and maybe I just didn't get the details quite right?

Reply:
Yea, I loved Enum's in my Java days, but can't really get the hang of them in Python...

Reply:
Yeah the Python enum feels super pointless, I guess if we're typing stuff really carefully it could be helpful but we're not, so...

We're not using it elsewhere so I think it's best not to here, either.